storages/python/README.md

```diff
@@ -60,3 +60,67 @@ Python storage plugins provided with DLite can be found in the
 
 See the [YAML plugin](python-storage-plugins/yaml_plugin.py) for a
 simple example of a working storage plugin.
+
+
+
+Plugins
+=======
+
+yaml
+----
+Generic plugin for reading and writing to file in YAML format.
+
+
+bson
+----
+Generic plugin for reading and writing to file in BSON format.
+
+
+pyrdf
+-----
+Generic plugin for reading and writing to file in RDF format.  It uses
+rdflib. The plugin is called pyrdf to not confuse with the rdf plugin
+implemented in C (using Redland librdf).
+
+
+csv
+---
+Read and write to file in tabular CSV format.  The metadata for CSV
+files should have only one dimension shared by all properties.
+
+
+blob
+----
+Specialised plugin for reading and writing a binary blob to file.  The
+content is specified using the metadata
+http://onto-ns.com/meta/0.1/Blob.
```

Comment:
This does not exist. Can you add it and make  link to it on the dlite-repo for now?